src/AutoAffix.js

```diff
@@ -32,6 +32,11 @@ class AutoAffix extends React.Component {
     this._windowScrollListener = addEventListener(
       ownerWindow(this), 'scroll', () => this.onWindowScroll()
     );
+
+    this._windowResizeListener = addEventListener(
```

Comment:
Should probably add something like this to `<Affix>` as well. I thought about doing this originally - it's one of those things that's the right thing like 90% of the time, but then you end up needing something much heavier like https://github.com/souporserious/react-measure to handle the rest of the cases. But this might be the best option - the `MutationObserver` polyfill for IE9/10 sorta sucks.